<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindow</class>
 <widget class="QMainWindow" name="AuthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 245, 245);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="login">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>801</width>
        <height>601</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="loginForm">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:rgb(255, 255, 255);
	background-color: rgb(38, 50, 56);
}

QPushButton:hover {
	background-color: rgb(42, 74, 74);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>100</y>
            <width>171</width>
            <height>71</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>36</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(38, 50, 56);</string>
          </property>
          <property name="text">
           <string>SIGN IN</string>
          </property>
         </widget>
         <widget class="QWidget" name="formLayoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>230</y>
            <width>441</width>
            <height>132</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>30</number>
           </property>
           <item row="0" column="1">
            <widget class="QLineEdit" name="SIusernameEntry">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid lightgrey; border-radius:5px; background-color: palette(base);
padding-left: 10px;
padding-right: 10px;</string>
             </property>
             <property name="maxLength">
              <number>32766</number>
             </property>
             <property name="placeholderText">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="SIpasswordEntry">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid lightgrey; border-radius:5px; background-color: palette(base);
padding-left: 10px;
padding-right: 10px;</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Password</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="SIerror">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>170</y>
            <width>441</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(245, 105, 101);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is error&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="signIn_5">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>420</y>
            <width>360</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 10px;
background-color: #3E6E6E;
}


QPushButton:hover {
	background-color: rgb(53, 94, 94);
}

QPushButton:pressed {
	background-color: rgb(31, 56, 56);
}</string>
          </property>
          <property name="text">
           <string>Sign In</string>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="RegisterNav">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(194, 214, 214);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>60</y>
            <width>301</width>
            <height>261</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resource.qrc">:/resources/images/exam.svg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>370</y>
            <width>181</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>28</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>New here?</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>420</y>
            <width>241</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Sign up and discover a great amount of new opportunities!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="signupNav">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>490</y>
            <width>201</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 10px;
border: 2px solid white;
background-color: transparent;
color: white;
}


QPushButton:hover {
	background-color: white;
color:  rgb(42, 74, 74);
}

QPushButton:pressed{
	background-color: rgb(240, 240, 240);
}</string>
          </property>
          <property name="text">
           <string>Sign Up &gt;</string>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="register_2">
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>811</width>
        <height>611</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="RegisterNav_2">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(194, 214, 214);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>60</y>
            <width>301</width>
            <height>261</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resource.qrc">:/resources/images/book.svg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>370</y>
            <width>261</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>28</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>One of us?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>420</y>
            <width>241</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>If you already have your own account, just sign in</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="signInNav">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>490</y>
            <width>201</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 10px;
border: 2px solid white;
background-color: transparent;
color: white;
}


QPushButton:hover {
	background-color: white;
color:  rgb(42, 74, 74);
}

QPushButton:pressed {
	background-color: #f0f0f0;
}</string>
          </property>
          <property name="text">
           <string>&lt; Sign In</string>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="loginForm_2">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color:rgb(255, 255, 255);
	background-color: rgb(38, 50, 56);
}

QPushButton:hover {
	background-color: rgb(42, 74, 74);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>191</width>
            <height>71</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>36</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(38, 50, 56);</string>
          </property>
          <property name="text">
           <string>SIGN UP</string>
          </property>
         </widget>
         <widget class="QWidget" name="formLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>170</y>
            <width>441</width>
            <height>231</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_4">
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>30</number>
           </property>
           <item row="0" column="1">
            <widget class="QLineEdit" name="SUusernameEntry_5">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid lightgrey; border-radius:5px; background-color: palette(base);
padding-left: 10px;
padding-right: 10px;</string>
             </property>
             <property name="maxLength">
              <number>32766</number>
             </property>
             <property name="placeholderText">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="SUpasswordEntry">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid lightgrey; border-radius:5px; background-color: palette(base);
padding-left: 10px;
padding-right: 10px;</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="SUpasswordEntry_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid lightgrey; border-radius:5px; background-color: palette(base);
padding-left: 10px;
padding-right: 10px;</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Confirm Password</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="SUerror">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>441</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(245, 105, 101);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is error&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="signIn_4">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>470</y>
            <width>360</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 10px;
background-color: #3E6E6E;
}


QPushButton:hover {
	background-color: rgb(53, 94, 94);
}

QPushButton:pressed {
	background-color: rgb(31, 56, 56);
}</string>
          </property>
          <property name="text">
           <string>Sign Up</string>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
